odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/service/DataEntityServiceImpl.java

```diff
@@ -153,6 +153,13 @@ public Mono<List<DataEntityDimensionsDto>> getDimensions(final Collection<String
             .flatMap(this::enrichParentGroups);
     }
 
+    @Override
+    public Mono<List<DataEntityDimensionsDto>> getDimensionsByIds(final Set<Long> ids) {
+        return reactiveDataEntityRepository.getDimensionsByIds(ids)
+            .flatMap(this::enrichEntityClassDetails)
+            .flatMap(this::enrichParentGroups);
```

Comment:
Don't we need `collectList` here?
```suggestion
            .flatMap(this::enrichParentGroups)
            .collectList();
```

Reply:
this::enrichParentGroups will return Mono<List<DataEntityDimensionsDto>>, so  there is no need to call  collectList()